### ISSUE TITLE
UI suggestions

### ISSUE THREAD

**Issue by Rexogamer** [CodeHubApp/CodeHub#609]
- add a specific edit button, as the button currently used for accessing editing looks like a share button
- the user avatar is weirdly stretched on the sidebar
- generally organise the sidebar better

(btw if there’s a CodeHub TestFlight I’d happily help test!)

**Comment by garrettw** [comment]
Just FYI, the "share" button, while typically used for that function, is not considered as such by Apple officially. They call it an "action" button. [See Dillon's comment here.](https://github.com/CodeHubApp/CodeHub/issues/291#issuecomment-312428425)

**Comment by Rexogamer** [comment]
I see. Either way, I feel a pencil would do much better for editing, and a triple-dot for the rest.

**Comment by steakknife** [comment]
Agreed. This was very confusing UX that doesn't follow customary conventions.
- Share should be share and only share.
- Pencil should be edit.
- Dots for the rest.

**Comment by naingwynn88** [comment]
CodeHub/Properties/AssemblyInfo.cs
